notebooks/wep_pipeline_timing.ipynb

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
<p>Maybe some additional explanation saying "you will need to change the repo directory if you want to access the timing of the pipeline in the summit or usdf repos. Change the path accordingly" or something like this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>